### ISSUE TITLE
Bug: the direction of the crafting table seems to be inverse

### ISSUE THREAD

**Issue by SepSol** [Tfarcenim/crafting-station#24]
When I place a Crafting Station and I put some block on top slots of the recipe, instead of showing far from me on the table, they show up near me. In other word the table's directional feature seems to have been rotated 180 degrees. Take a look at these screenshots:

![image](https://user-images.githubusercontent.com/47871887/67165369-d7a0f180-f390-11e9-9b4a-9afff586eb01.png)
![image](https://user-images.githubusercontent.com/47871887/67165359-c5bf4e80-f390-11e9-980c-ee34325c07da.png)

**Comment by SepSol** [comment]
That's solved for the recipes thanks :)

**Comment by SepSol** [comment]
I want to re-open the issue but I don't know how. I found something new about the directions. The directions seem to be ok but the items are flipped on the Crafting Station (on both the full-block version and the half-block version). Below are some screenshots explaining the issue:

![image](https://user-images.githubusercontent.com/47871887/68007439-e04acf00-fc90-11e9-9fe0-ef407ac7549c.png)
![2019-11-01_10 19 27](https://user-images.githubusercontent.com/47871887/68007888-42f09a80-fc92-11e9-8cad-d71bb0256dd5.jpg)

I crafted the top picture in all 4 directions and here's the result in each one of them.
